Doc/taipy#260 - taipy run CLI (#548)
Avaiga/taipy-doc: docs/manuals/cli/manage-versions.md

## List capabilities with the --help option

> To display the help with all command line arguments, you can run `taipy help manage-versions` command.

Reviewer: I think that's wrong.
"To display the help with all command line arguments", you certainly don't run "taipy help manage-versions"

However, to get help on the 'manage-versions' command, you can do that indeed.

Author: We support both actually

Author: ```suggestion
To display the help message of the `manage-version` command, you can run `taipy help manage-versions` command.
```
